exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -103,19 +106,26 @@ func newPRWExporter(cfg *Config, set exporter.Settings) (*prwExporter, error) {
 		return nil, err
 	}
 
+	err = config.RemoteWriteProtoMsg.Validate(cfg.RemoteWriteProtoMsg)
```

Comment:
We should validate configuration in a Validate() function on the Config struct.  See https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-collector-contrib%20Validate&type=code for other examples.

Reply:
fixed